firefox-ios/Client/Frontend/Autofill/Address/Edit/EditAddressViewController.swift

```diff
@@ -48,27 +34,28 @@ class EditAddressViewController: UIViewController, WKNavigationDelegate, WKScrip
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        listenForThemeChange(view)
+    }
 
+     override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
         setupWebView()
-        setupActivityIndicator()
-        listenForThemeChange(view)
     }
 
-    private func setupActivityIndicator() {
-        self.view.addSubview(activityIndicator)
-        NSLayoutConstraint.activate([
-            activityIndicator.centerXAnchor.constraint(equalTo: view.centerXAnchor),
-            activityIndicator.centerYAnchor.constraint(equalTo: view.centerYAnchor)
-        ])
+    override func viewWillDisappear(_ animated: Bool) {
```

Comment:
I think a more appropriate method to do this is viewDidDissappear. In this way we know that we no longer need it.

Reply:
Yep. Good call.

Reply:
Updated in https://github.com/mozilla-mobile/firefox-ios/pull/20738/commits/9e98f326ab818d5529c19f91c0a4e2464b67bac0